src/test/java/org/elasticsearch/test/ElasticsearchIntegrationTest.java

```diff
@@ -227,9 +226,25 @@
  * </p>
  */
 @Ignore
-@AbstractRandomizedTest.Integration
+@ElasticsearchIntegrationTest.Integration
+@LuceneTestCase.SuppressFileSystems("ExtrasFS") // doesn't work with potential multi data path from test cluster yet
 public abstract class ElasticsearchIntegrationTest extends ElasticsearchTestCase {
```

Comment:
I opened an issue: #10659

If we do it here and someone touches these classes before we merge this, git shits a brick and I don't want this change blocked by that.